sanic_ext/extensions/openapi/ui/swagger.html

```diff
@@ -15,8 +16,9 @@
 
     <body>
         <div id="openapi">
-            <script src="//pagecdn.io/lib/swagger-ui/latest/swagger-ui-bundle.js"></script>
-            <script src="//pagecdn.io/lib/swagger-ui/latest/swagger-ui-standalone-preset.js"></script>
+            <script src="//cdnjs.cloudflare.com/ajax/libs/swagger-ui/__VERSION__/swagger-ui-bundle.js"></script>
+            <script
+                src="//cdnjs.cloudflare.com/ajax/libs/swagger-ui/__VERSION__/swagger-ui-standalone-preset.js"></script>
```

Comment:
Do we want to use minified versions for this too, maybe?

Reply:
Well, when I looked I was certain there weren't any. But I looked again and now I see them :joy:

Reply:
✨ _MAGIC_ ✨